org.eclipse.jdt.core.javac/src/org/eclipse/jdt/internal/javac/dom/JavacTypeBinding.java

```diff
@@ -612,7 +612,10 @@ public boolean isLocal() {
 
 	@Override
 	public boolean isMember() {
-		return this.typeSymbol.owner instanceof ClassSymbol;
+		if (isClass() || isInterface() || isEnum()) {
```

Comment:
what about records?

Reply:
Reproducer for where it's needed, I'll file this upstream:

aaa/MyClass.java
```java
package aaa;

public MyClass implements MyInterface {
}
```

bbb/MyInterface.java
```java
package bbb;

import bbb.Namespace.MyRecord;

public interface MyInterface {
    MyRecord getMyRecord();
}
```

bbb/Namespace.java
```java
package bbb;

public class Namespace {
    private Namespace() {}

    public static record MyRecord(double field1, double field2) {}
}
```

Then, in MyClass.java, apply the quickfix to add the unimplemented methods. Without the `isRecord()` check, the import for `MyRecord` will not be added until you organize imports.